contributors/guide/review-guidelines.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: "Pull Request Review Guidelines"
 weight: 13
-slug: "review-guidelines"
+description: |
+  A collection of of tips to help both those looking to get their PR reviewed
+  and the code reviewers themselves.
```

Comment:
(In a different PR) We should link expectations.md in this doc since they are related.

Reply:
Yeah...there's a lot of stuff that needs to be deduped and link at the moment >_>